kdl_parser/cmake/FindTinyXML2.cmake

```diff
@@ -0,0 +1,74 @@
+##################################################################################################
+# 
+# CMake script for finding TinyXML2.
```

Comment:
Just noting it looks like this was copied from https://github.com/ros/cmake_modules/blob/0.4-devel/cmake/Modules/FindTinyXML2.cmake

Reply:
Yes, I copied it from the `cmake_modules` repo to be used as standalone. I mentioned the source in the git commit. Should I leave a note in the CMakeLists.txt file where the cmake modules are from?